Projects/surface_photometry.ipynb

```diff
@@ -0,0 +1,1332 @@
+    "    \"\"\"\n",
+    "    Create an RGB color composite image.\n",
+    "    \n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    image : `MultibandImage`\n",
+    "        `MultibandImage` to display.\n",
+    "    bgr : sequence\n",
+    "        A 3-element sequence of filter names (i.e. keys of the exps dict) indicating what band\n",
+    "        to use for each channel. If `image` only has three filters then this parameter is ignored\n",
+    "        and the filters in the image are used.\n",
+    "    stretch: int\n",
+    "        The linear stretch of the image.\n",
+    "    Q: int\n",
+    "        The Asinh softening parameter.\n",
+    "    scale: list of 3 floats, each less than 1. (default: None)\n",
+    "        Re-scales rgb channels  \n",
+    "        \n",
+    "    Returns\n",
+    "    -------\n",
+    "    rgb: ndarray \n",
+    "        RGB (integer, 8-bits per channel) colour image as an NxNx3 numpy array.\n",
+    "    \"\"\"\n",
+    "    \n",
+    "    # If the image only has 3 bands, reverse the order of the bands to produce the RGB image\n",
+    "    if len(image) == 3:\n",
+    "        bgr = image.filters\n",
+    "    # Extract the primary image component of each Exposure with the .image property, and use \n",
+    "    # .array to get a NumPy array view.\n",
+    "\n",
+    "    if scale is None:\n",
+    "        \n",
+    "        r_im = image[bgr[2]].array # numpy array for the r channel\n",
+    "        g_im = image[bgr[1]].array # numpy array for the g channel\n",
+    "        b_im = image[bgr[0]].array # numpy array for the b channel\n",
+    "    \n",
+    "    else:\n",
+    "        ### manually re-scaling the images here \n",
+    "        r_im = image[bgr[2]].array*scale[0]\n",
+    "        g_im = image[bgr[1]].array*scale[1]\n",
+    "        b_im = image[bgr[0]].array*scale[2]     \n",
+    "    \n",
+    "    rgb = make_lupton_rgb(image_r=r_im,  \n",
+    "                          image_g=g_im,  \n",
+    "                          image_b=b_im,  \n",
+    "                          stretch=stretch, Q=Q)  # parameters used to stretch and scale the pixel values\n",
+    "    \n",
+    "    return rgb          "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **Part 1: Selecting a galaxy by coordinates**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As detailed in other tutorials, first we need to set up a Butler based on the parameters we obtained in the previous section.  We do this by galaxy coordinates, looping through available imaging repositories (DEEP which includes DEEP and ULTRA-DEEP tracts and WIDE). We selected this repository as it is the most complete dataset of HSC PDR2 in the LSP at the time of writing this notebook [(more info)]( https://confluence.lsstcorp.org/display/DM/S20+HSC+PDR2+Reprocessing). Thank you Yusra AlSayaad for the much needed help with this!\n",
```

Comment:
I'm missing what were the "parameters we obtained in the previous section"? Was this an RA,DEC?

Reply:
Gosh, I don't know.  Did I write this section?  We could probably just remove 'based on the parameters we obtained in the previous section', because as far as I can tell the only parameter we obtained there is the Butler import itself.

Reply:
Maybe @nchamba knows? Maybe this means the repo path?

Reply:
yes I agree, i think it's a mistake, I think it should be removed and replaced with "Butler to load a given dataset" or something equivalent....